examples/src/components/BucketExecutorExample.jsx

```diff
@@ -0,0 +1,250 @@
+// (C) 2007-2019 GoodData Corporation
+/* eslint-disable react/jsx-closing-tag-location */
+import React, { Component } from "react";
+import { BucketExecutor, LoadingComponent, ErrorComponent, Model } from "@gooddata/react-components";
+
+import { totalSalesIdentifier, projectId, locationStateDisplayFormIdentifier } from "../utils/fixtures";
+
+const presets = {
+    kpi: {
+        props: {
+            dimensions: [[Model.measure(totalSalesIdentifier)]],
+        },
+        label: "KPI",
+        // eslint-disable-next-line react/prop-types
+        DataComponent: ({ result }) => {
+            return (
+                <p
+                    className="s-bucket-executor-kpi"
+                    style={{
+                        height: 60,
+                        margin: "10px 0",
```

Comment:
Nitpick: many of the styles are shared across the file (`fontWeight` among others). Maybe create someting like `commonStyles`?